client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -1,11 +1,18 @@
 @import "@wordpress/components/build-style/style";
 @import "@automattic/onboarding/styles/mixins";
-@import "@wordpress/base-styles/_breakpoints.scss";
-@import "@wordpress/base-styles/_mixins.scss";
+@import "@wordpress/base-styles/breakpoints.scss";
+@import "@wordpress/base-styles/mixins.scss";
 
 $gray-100: #101517;
 $gray-60: #50575e;
 $design-button-primary-color: rgb(17, 122, 201);
+$break-design-preview: 1024px;
+
+@mixin break-design-preview() {
```

Comment:
It might be better if we could reuse `break-design-preview` from `@automattic/design-preview`